src/sentry/rules/processing/delayed_processing.py

```diff
@@ -456,13 +457,10 @@ def apply_delayed(project_id: int, *args: Any, **kwargs: Any) -> None:
     rules_to_groups = get_rules_to_groups(rulegroup_to_event_data)
 
     # STEP 3: Fetch the Rule models we need to check
-    alert_rules_qs = Rule.objects.filter(id__in=list(rules_to_groups.keys()))
-    snoozed_rules = set(
-        RuleSnooze.objects.filter(rule__in=alert_rules_qs, user_id=None).values_list(
-            "rule", flat=True
-        )
+    alert_rules_queryset = Rule.objects.filter(id__in=list(rules_to_groups.keys())).exclude(
+        id__in=Subquery(RuleSnooze.objects.filter(rule_id=OuterRef("id")).values("rule_id"))
```

Comment:
Wait josh no lol the `user_id=None` is from the original query, we still need that.

Reply:
the lack of a `user_id` in this table means that it's snoozed for _everyone_ as opposed to being snoozed for an individual

Reply:
lol 😅 - i'll add it back.

just curious, what case we would have for that?